packages/amplify-category-api/src/__tests__/provider-utils/awscloudformation/ecs-apigw-stack.test.ts

```diff
@@ -19,7 +19,7 @@ describe('ecs stack', () => {
           undefined,
           undefined,
           undefined,
-          undefined,
+          'testImage',
```

Comment:
why is this needed ?

Reply:
CDK added validation for that property to be present. Sigh... another technically breaking change.